engines/enginetest/testutils.go

```diff
@@ -27,6 +30,12 @@ func evalNilOrPanic(f func() error, a ...interface{}) {
 	nilOrpanic(f(), a...)
 }
 
+func assert(condition bool, a ...interface{}) {
+	if !condition {
+		fmtPanic(a...)
+	}
+}
+
```

Comment:
I think @gregarndt has been using an assert library for this - we should probably align on either using our own or using this. Since assertions are not useful only for engine tests, this should probably go somewhere outside the engine test package.